docs/HACKING.md

```diff
@@ -163,90 +258,149 @@ If the bundle channel needs to be changed, the `CHANNELS` and/or `DEFAULT_CHANNE
 
-#### Pre-requisites
-[opm](https://github.com/operator-framework/operator-registry/) has been installed on the build system.
-All previous [pre-requisites](#pre-requisites) must be satisfied as well.
+See the OpenShift docs found
+[here](https://docs.openshift.com/container-platform/latest/operators/admin/olm-managing-custom-catalogs.html#olm-creating-catalog-from-index_olm-managing-custom-catalogs)
 
-You can skip this step if you want to run the operator for [developer testing purposes only](#development-workflow)
+### Follow the normal procedure to deploy an operator
 
-An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
-cluster.
+See the OpenShift docs found
+[here](https://docs.openshift.com/container-platform/latest/operators/admin/olm-adding-operators-to-cluster.html)
+
+## Test
+
+If you are not running e2e tests, it is likely that you will want to deploy a custom workload to test your changes with.
+Find more information [here](custom-workload.md)
+
+## Cleanup
+
+### Cleaning up a manual deployment  
+
+To remove the installed resources:
 
 ```shell script
-opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$INDEX_TAG --container-tool podman
+make undeploy
 ```
 
-You should then push the newly created index image to the remote repository:
+## Cleaning up an OLM deployment
+
 ```shell script
-podman push $INDEX_REPOSITORY:$INDEX_TAG
+hack/olm.sh cleanup
 ```
 
-#### Editing an existing operator index
-An existing operator index can have bundles added to it:
+## Cleaning up e2e tests
+
+#### Delete Machinesets
+
 ```shell script
-opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
+oc delete machineset e2e -n openshift-machine-api
+oc delete machineset e2e-wm -n openshift-machine-api
+oc delete configmap windows-instances -n openshift-windows-machine-config-operator
 ```
-and removed from it:
+
+#### Stash changes
+
+Running e2e tests and deploying the OLM both make changes to the files
+`bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml` and
+`operator-sdk`. Stash changes to those files.
+
+#### Run the cleanup OLM script
+
+clean up the operator lifecycle manager created by the e2e tests by running the hack script
+
 ```shell script
-opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
+hack/olm.sh cleanup
+oc delete ns wmco-test
 ```
 
-## Updating Git submodules
+## Updating submodule versions
+
 This project contains git submodules for the following components:
+
 - kubernetes
 - ovn-kubernetes
 - containernetworking-plugins
 - promu
 - windows_exporter
 
-To update git submodules you can use the script hack/update_submodules.sh
+To update the versions of the git submodules you can use the script hack/update_submodules.sh
 Use the help command for up to date instructions:
+
 ```shell script
 hack/update_submodules.sh -h
 ```
 
-Alternatively, it can be done manually via:
+## Creating a new operator index
```

Comment:
It seems odd you have this section here instead of where you are creating rest of the OLM artifacts.